noaa/gefs/forecast/reformat.py

```diff
@@ -352,6 +360,10 @@ def reformat_init_time_i_slices(
                     chunks = template.chunk_args(chunk_template_ds)
                     data_array.chunk(chunks).to_zarr(store, region="auto")
                     yield (data_var, max_lead_times)
+                # Reclaim space once done.
+                for coord_and_path in coords_and_paths:
+                    if coord_and_path[1] is not None:
+                        coord_and_path[1].unlink()
```

Comment:
nit for readability:
```suggestion
                # Reclaim space once done.
                for coord, filepath in coords_and_paths:
                    if filepath is not None:
                        filepath.unlink()
```